tests/integration/features/tabs.feature

```diff
@@ -635,6 +635,16 @@ Feature: Tab management
         Then the following tabs should be open:
             - data/hello.txt (active)
 
+    Scenario: Double-undo with single tab on last-close default page
+        Given I have a fresh instance
+        When I open about:blank
+        And I set tabs -> last-close to default-page
+        And I set general -> default-page to about:blank
+        And I run :undo
```

Comment:
The issue started with the first undo. It should've done nothing, but instead obliterated the only open tab, leaving you with no webview. This test makes sure that doesn't happen.
It would be nice to test undoing twice just to be sure, but I'm not sure how to expect the same error ('nothing to undo') twice.